homeassistant/components/renault/services.yaml

```diff
@@ -27,6 +27,43 @@ ac_cancel:
         device:
           integration: renault
 
+ac_set_schedules:
+  fields:
+    vehicle:
+      required: true
+      selector:
+        device:
+          integration: renault
+    schedules:
+      example: >-
+        [
+          {
+            "id": 1,
+            "activated": false
+          },
+          {
+            "id": 2,
+            "activated": true,
+            "monday": { "readyAtTime": "T20:45Z" },
+            "sunday": { "readyAtTime": "T20:45Z" }
+          },
+          {
+            "id": 3,
+            "activated": false
+          },
+          {
+            "id": 4,
+            "activated": false
+          },
+          {
+            "id": 5,
+            "activated": false
+          }
+        ]
```

Comment:
We should not create services that require the user to put in JSON objects. This is too error prone imo

Reply:
I changed both examples in the file. Also updated the documentation.